addons/rearm/script_component.hpp

```diff
@@ -19,6 +19,7 @@
 
 
 #define REARM_ACTION_DISTANCE 7
+#define REARM_ACTION_DISTANCE_SQR REARM_ACTION_DISTANCE ^ 2
```

Comment:
You are now squaring the number at runtime every time... It doesn't matter no need to fix.